atlasaction/action.go

```diff
@@ -833,15 +840,29 @@ func (a *Actions) MonitorSchema(ctx context.Context) error {
 	}); err != nil {
 		return fmt.Errorf("failed to login to Atlas Cloud: %w", err)
 	}
-	res, err := a.Atlas.SchemaInspect(ctx, &atlasexec.SchemaInspectParams{
+	params := &atlasexec.SchemaInspectParams{
 		URL:     db.String(),
 		Schema:  id.Schemas,
 		Exclude: id.Schemas,
-		Format:  `{{ printf "# %s\n%s" .Hash .MarshalHCL }}`,
-	})
+		Format:  `{{ printf "# %s\n%s\n%s" .RedactedURL .Hash .MarshalHCL }}`,
```

Comment:
Mark it as the comment
```suggestion
		Format:  `{{ printf "# %s\n# %s\n%s" .RedactedURL .Hash .MarshalHCL }}`,
```

Reply:
@giautm not sure I understand you, why use # as delimiter ?